Library/Formula/python3.rb

```diff
@@ -50,10 +50,16 @@ class Python3 < Formula
   # X11.
   patch :DATA if build.with? "brewed-tk"
 
+  def lib_cellar
+    if OS.mac?
+      prefix/"Frameworks/Python.framework/Versions/2.7/lib/python2.7"
+    else
+      prefix/"lib/python#{xy}/site-packages"
+    end
+  end
+
```

Comment:
Please use identical code to `python.rb`. It'll make maintenance of the two formulae easier.
See https://github.com/Homebrew/linuxbrew/blob/master/Library/Formula/python.rb#L56-L59